.idea/.name

```diff
@@ -0,0 +1 @@
+LoginTest.java
```

Comment:
files inside .idea folder is auto-generate files by building, please add them in git ignore and do not push it

Reply:
i have updated for it